SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -480,6 +491,77 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
   }
 }
 
+
+void CTurbSSTSolver::SetNuTilde_WF(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics,
+                                  CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
+
+  const unsigned short max_iter = 100;
+
+  /* --- tolerance has LARGE impact on convergence, do not increase this value! --- */
+  const su2double tol = 1e-12;
```

Comment:
I won't 🤞 ;)

Reply:
1e-10 was the the original value and if you look at the residual plots, changing to 1e-12 was a big improvement!

Reply:
Maybe document those values there in the comment as well. just for safety